website_sale_product_attachment/models/ir_attachment.py

```diff
@@ -13,3 +14,7 @@ class IrAttachment(models.Model):
         help="Attachment publicly downladable from eCommerce pages "
         "in these product templates.",
     )
+    website_name = fields.Char(
```

Comment:
I think the field name is confusing, because it seems to indicate the name of the website, and not the name of the attachment when displayed in the website. Even more when the field is displayed just next to the website dropdown in the backend:

![imagen](https://user-images.githubusercontent.com/973709/134884829-b08151e6-6e39-4014-90a6-cf9b77de60a7.png)


Also, the attachment name probably needs to be translated. If you're browsing the site in English you might want to download the "Instructions manual" while when browsing in Spanish, you want the "Manual de instrucciones".

So this would be better IMHO:

```suggestion
    website_name = fields.Char(
        string="Name in website",
        translate=True,
```

Reply:
String changed in the next push to `Name in e-commerce` for being more specific, but the translatable property is another thing, as this will mean that sorting will depend on this translation. As our use case doesn't involve translations for now, I have added it to the ROADMAP for being handled later if ever needed.